lib/plaid/products/asset_report.rb

```diff
@@ -10,7 +10,7 @@ class AssetReport < BaseProduct
     #                  the Asset Report. Plaid will return as much data as
     #                  possible within the implied date range, up to a maximum
     #                  of 730 days (2 years) of history.
-    # options -        An optional object containing client-provided user
+    # options        - An optional object containing client-provided user
```

Comment:
Let's make `options` really optional by specifying `, options={}` in the method definition.

Reply:
⚡️